### PR TITLE
Occupation comp

### PR DESCRIPTION
**Pull request by mb556**
created a component that only exports the name, level, overview and technicalLevelName which required the update of OccupationCard.jsx and OccupationCard.stories.js

### CODE REVIEW

File: frontend/src/components/occupationcard.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+export default function OccupationCard({ occupation }) {
+  if (!occupation) {
+    return <div>Loading...</div>;
+  }
+
+  return (
+    <div className="occupation-card">
+      <h2>{occupation.name} (Code: {occupation.stdCode})</h2>
```
Comment (camelPhonso) [inline frontend/src/components/occupationcard.jsx:10]: let's remove the occupation stdCode from here, the user doesn't need to see that

File: frontend/src/components/occupationcard.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      <p><strong>Status:</strong> {occupation.statusName} (Level {occupation.level}, Version {occupation.versionNo})</p>
+      <p><strong>Overview:</strong> {occupation.overview}</p>
+      <div>
+        <strong>Summary:</strong>
+        <div dangerouslySetInnerHTML={{ __html: occupation.summary }} />
```
Comment (camelPhonso) [inline frontend/src/components/occupationcard.jsx:15]: what did we say about pushing code we don't understand? why don't we just interpolate the summary into the code like we're doing with the other values here?

File: frontend/src/components/occupationcard.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      </div>
+      <div>
+        <strong>Map Hierarchy:</strong>
+        <ul>
+          <li><strong>Route:</strong> {occupation.mapHierarchy.routeName} (ID: {occupation.mapHierarchy.routeId})</li>
+          <li><strong>Pathway:</strong> {occupation.mapHierarchy.pathwayName} (ID: {occupation.mapHierarchy.pathwayId})</li>
```
Comment (camelPhonso) [inline frontend/src/components/occupationcard.jsx:21]: As a rule we don't want to be exposing Id values unless they specifically are meant for the end user.

**Review by camelPhonso — CHANGES_REQUESTED** [review]
This is good stuff but you didn't read the issue description when you were working on it. 

There's a lot of information being passed onto the component that wasn't asked for, so let's go back and read the description in the issue and strip down this code to what's necessary

File: frontend/src/components/occupationcard.jsx (hunk 3 above)
Comment (mb556) [reply frontend/src/components/occupationcard.jsx]: okay that has been moved off

**Review by camelPhonso — APPROVED** [review]
Good job. Let's connect this up to the rest of the app !

**Pull request merged**